package.json

```diff
@@ -7,7 +7,10 @@
     "dev": "nodemon --watch '/src/**/*.ts' --exec \"ts-node\" src/index.ts",
     "build": "tsc",
     "test": "jest",
-    "start": "node dist/app.js"
+    "start": "node dist/app.js",
+    "db:start": "docker run --name some-postgres --rm -e POSTGRES_PASSWORD=postgres -p 5432:5432 -d postgres",
+    "db:stop": "docker stop some-postgres",
+    "db:init": "ts-node src/db-init.ts"
```

Comment:
Added some npm scripts to help with manually start/init/stop postgres database with some short commands.